packages/nova-react/src/graphql/types.ts

```diff
@@ -82,11 +82,17 @@ export type PaginationFn = (
   options?: RefetchOptions,
 ) => Disposable;
 
-type FetchPolicy =
+export type FetchPolicy =
   | "store-or-network"
   | "store-and-network"
   | "network-only"
-  | "store-only";
+  | "store-only"
+  | "cache-first"
```

Comment:
What are the new fetch policies? Are there from apollo-client? I think we should just use Relay ones as [duct tape does translation](https://github.com/microsoft/graphitation/blob/main/packages/apollo-react-relay-duct-tape/src/convertFetchPolicy.ts). Right, @alloy ?

Reply:
Apollo fetch policies. If we don't need to support them, I'm happy to remove them

Reply:
Yes, only the Relay ones, and specifically the ones captured in the duct-tape transform 👍